scripts/ci-entrypoint.sh

```diff
@@ -58,10 +58,26 @@ setup() {
     if [[ -z "${CLUSTER_TEMPLATE:-}" ]]; then
         select_cluster_template
     fi
+    if [[ -n "${CI_VERSION:-}" ]] || [[ -n "${USE_CI_ARTIFACTS:-}" ]] || [[ "${KUBERNETES_VERSION:-}" =~ "latest" ]]; then
+        GOPATH="$(go env GOPATH)"
+        if ls "${GOPATH}"/src/k8s.io/kubernetes; then
```

Comment:
#2325 has merged, this should be good to go after rebase

Reply:
@lzhecheng thank you for your patience ❤️